ted/src/main/java/frc/robot/commands/ShooterSetAngleWithVisionCommand.java

```diff
@@ -0,0 +1,76 @@
+// ************************************************************
+// Bishop Blanchet Robotics
+// Home of the Cybears
+// FRC - Crescendo - 2024
+// File: ShooterSetAngleWithVisionCommand.java
+// Intent: Wraps the ShooterSetAngleWithVision command to set angle with vision
+// ************************************************************
+
+// ʕ •ᴥ•ʔ ʕ•ᴥ•  ʔ ʕ  •ᴥ•ʔ ʕ •`ᴥ´•ʔ ʕ° •° ʔ ʕ •ᴥ•ʔ ʕ•ᴥ•  ʔ ʕ  •ᴥ•ʔ ʕ •`ᴥ´•ʔ ʕ° •° ʔ 
+
+package frc.robot.commands;
+
+import frc.robot.common.DistanceMeasurement;
+import frc.robot.common.MotorUtils;
+import frc.robot.control.Constants;
+import frc.robot.subsystems.CameraSubsystem;
+import frc.robot.subsystems.ShooterAngleSubsystem;
+
+/**
+ * Forms a command to shoot the shooter
+ */
+public class ShooterSetAngleWithVisionCommand extends ShooterSetAngleCommand {
+
+  private double desiredAngleDegrees;
+  private ShooterAngleSubsystem shooterAngleSubsystem;
+  private CameraSubsystem cameraSubsystem;
+  private double farSlope = -4.65;
```

Comment:
curious to know what is going on here with these slopes and offsets ... not understanding from the code

Reply:
added explanation